scripts/deploy-diamond.ts

```diff
@@ -23,9 +23,7 @@ const createAppDiamondCuts = async (): Promise<DiamondCut[]> => {
   // Deploy Embalmer Facet
```

Comment:
Unrelated to this PR -- do we need the private keys library anymore (lines 14-16) since unencrypted shards are used for unwrapping?

Reply:
I don't think we need the private keys library anymore